packages/requestNetworkSmartContracts/contracts/core/RequestCore.sol

```diff
@@ -355,7 +362,7 @@ contract RequestCore is Administrable {
         constant
         returns(uint8)
     {
-        for (uint8 i = 0; i < 256 && subPayees[_requestId][i].addr != address(0); i = i.add(1)) {
+        for (uint8 i = 0; subPayees[_requestId][i].addr != address(0); i = i.add(1)) {
```

Comment:
Do we have guarantees this won't go into an infinite loop?
We could keep the `i` counter but having it as uint16

Reply:
the safeMath i.add(1) prevent from a infinite loop. the add will assert if i > 255.